wordpress-mobile/test-case-reminder-bot
# Override default configuration with in-repo config file

This PR adds a way to override default test-cases repository configuration, which will allow to reuse it in different projects

Testing Instructions:
- run unit tests :D 
- WIP

I also extracted helpers into a module, that should make testing a bit easier, but frankly, it is not needed for this PR. I'm happy to revert it back if desired.
